build.html

```diff
@@ -13,7 +13,7 @@
     <p>Verify that you are logged in. Try again by<br> pressing the <strong>Refresh</strong> button below.</p>
     <div class="btn refresh-chat" refresh-chat>Refresh</div>
   </div>
-  <div class="empty-area"><p>To start messaging contacts tap the <i class="fa fa-pencil-square-o"></i> at the top of your screen.</p></div>
+  <div class="empty-area"><p class="empty-area-information">To start messaging contacts tap the <i class="fa fa-pencil-square-o"></i> at the top of your screen.</p></div>
```

Comment:
Is this not caused by `.empty-area` having a `flex-flow: column nowrap;` CSS property assigned? It looks like it's the `nowrap` part that's preventing the line from wrapping. Perhaps we could just change that?

Reply:
@tonytlwu 
I've tested this and it's not a `nowrap` part that causes this but a `column` part.
Example of the behavior.
![pr79](https://user-images.githubusercontent.com/53430352/71618508-7edba980-2bc8-11ea-8233-bb101581cd76.gif)

May I remove `flex-flow` CSS property?
It looks like it won't affect any other browsers.

Reply:
@YaroslavOvdii I don't know enough to tell you whether you can remove that property. You're welcome to do it and test it with all modes (with and without messages, and on different platforms/devices/OS/versions).

Reply:
@tonytlwu 
Altered flex-flow to `flex-flow: row nowrap` and it looks okay.

MS IE11
![Chat IE](https://user-images.githubusercontent.com/53430352/71982643-a0bed700-322d-11ea-98e6-45f2fdd7c8ed.png)

Chrome
![Chat Chrome](https://user-images.githubusercontent.com/53430352/71982644-a0bed700-322d-11ea-98a6-39d51874de35.png)

MS Edge
![Chat Edge](https://user-images.githubusercontent.com/53430352/71982645-a1576d80-322d-11ea-8899-1900561a92d9.png)

Firefox
![Chat Firefox](https://user-images.githubusercontent.com/53430352/71982646-a1576d80-322d-11ea-9949-74aa5259123b.png)